### ISSUE TITLE
Accessibility: Using Keyboard in Navigation

### ISSUE THREAD

**Issue by simenandre** [pulumi/docs#11536]
### What happened?

While tabbing through the website (pulumi.com), you cannot open the mobile version of the navigation or open dropdown menus. There might be other things on the topic of accessibility.

Eager to help!

### Example

Go to https://pulumi.com and try to navigate with your keyboard.

### Output of `pulumi about`

N/A

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by cnunciato** [comment]
Indeed that has been fixed!
